Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -0,0 +1,1471 @@
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.PolicyAssignments(val.ID && val.ID === obj.ID)': context_entry
+    }
+    entry_white_list_count = switch_list_to_list_counter(context_entry)
+    presented_output = ['ID', 'Name', 'PolicyName', 'PolicyID', 'PolicyDescription', 'Targets']
+    human_readable = tableToMarkdown(title, entry_white_list_count, headers=presented_output)
+    return human_readable, context, raw_response
+
+
+def get_deployable_devices_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    limit = args.get('limit', 50)
+    offset = args.get('offset', 0)
+    raw_response = client.get_deployable_devices(limit, offset)
+    items = raw_response.get('items')
+    if items:
+        context_entry = [{
+            'CanBeDeployed': item.get('canBeDeployed', ''),
+            'UpToDate': item.get('upToDate', ''),
+            'DeviceID': item.get('device', {}).get('id', ''),
+            'DeviceName': item.get('device', {}).get('name', ''),
+            'DeviceType': item.get('device', {}).get('type', ''),
+            'Version': item.get('version', '')
+        } for item in items
+        ]
+        title = f'{INTEGRATION_NAME} - List of deployable devices:'
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.DeployableDevices(val.ID && val.ID === obj.ID)': context_entry
+        }
+        presented_output = ['CanBeDeployed', 'UpToDate', 'DeviceID', 'DeviceName', 'DeviceType', 'Version']
+        human_readable = tableToMarkdown(title, context_entry, headers=presented_output)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - Could not find any deployable devices.', {}, {}
+
+
+def get_device_records_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    limit = args.get('limit', 50)
+    offset = args.get('offset', 0)
+    raw_response = client.get_device_records(limit, offset)
+    items = raw_response.get('items')
+    if items:
+        context_entry = [{
+            'ID': item.get('id', ''),
+            'Name': item.get('name', ''),
+            'HostName': item.get('hostName', ''),
+            'Type': item.get('type', ''),
+            'DeviceGroupID': item.get('deviceGroup', {}).get('id', '')
+        } for item in items
+        ]
+        title = f'{INTEGRATION_NAME} - List of device records:'
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.DeviceRecords(val.ID && val.ID === obj.ID)': context_entry
+        }
+        presented_output = ['ID', 'Name', 'HostName', 'Type', 'DeviceGroupID']
+        human_readable = tableToMarkdown(title, context_entry, headers=presented_output)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - Could not find any device records.', {}, {}
+
+
+def deploy_to_devices_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    force_deploy = args.get('for
```

Comment:
```suggestion
    LOG(f'Command being called is {command}')
```

Reply:
What did you mean here?

Reply:
not sure why it commented on that line, but i meant to replace `LOG('command is %s' % (demisto.command(),))` with my suggestion